RUFAS/biophysical/manure/manure_nutrient_manager.py

```diff
@@ -0,0 +1,300 @@
+from __future__ import annotations
+
+import math
+from typing import Any
+
+from RUFAS.output_manager import OutputManager
+from RUFAS.data_structures.manure_nutrients import ManureNutrients
+from RUFAS.data_structures.manure_to_crop_soil_connection import NutrientRequest, NutrientRequestResults
+from RUFAS.data_structures.manure_types import ManureType
+
+
+class ManureNutrientManager:
+    def __init__(self) -> None:
+        """Initialize the manure nutrient manager."""
+        self.om = OutputManager()
+
+        self.nutrients_by_manure_category: [ManureType, ManureNutrients] = {
+            ManureType.LIQUID: ManureNutrients(manure_type=ManureType.LIQUID),
+            ManureType.SOLID: ManureNutrients(manure_type=ManureType.SOLID),
+        }
+
+    def get_values(self, manure_type: ManureType) -> ManureNutrients:
+        """
+        Get the current nutrient values stored in the manager by manure type.
```

Comment:
```suggestion
        Get the current nutrient values stored in the nutrient manager by manure type.
```
I don't know if this is necessary but just specify this is the manure nutrient manager and not the manure manager